.gitignore

```diff
@@ -15,6 +15,6 @@
 /documentation/.psci*
 
 /documentation/html/test.js
-./spago
+/.spago
```

Comment:
Can't this be just `.spago/`?

Reply:
Indeed, I think `.spago/` or even just `.spago` would work. With the leading `/`, I believe a file/directory named `.spago` located in a subdirectory _would not_ be ignored, so maybe `.spago` would be the best pattern to use... What do you think @JordanMartinez ?

Reply:
I'd go with `.spago/`

Reply:
I believe there is a template .gitignore file in the PureScript-contrib governance repo that can be used here

Reply:
Fair enough @thomashoneyman... I'll open a new PR.